integration-tests/src/tests/features/access_key_nonce_for_implicit_accounts.rs

```diff
@@ -198,7 +198,9 @@ fn get_status_of_tx_hash_collision_for_near_implicit_account(
 /// Test that duplicate transactions from NEAR-implicit accounts are properly rejected.
 #[test]
 fn test_transaction_hash_collision_for_near_implicit_account_fail() {
-    let protocol_version = ProtocolFeature::AccessKeyNonceForImplicitAccounts.protocol_version();
+    #[allow(deprecated)]
+    let protocol_version =
```

Comment:
Remove variable `protocol_version` and change call below to the following:
```
        get_status_of_tx_hash_collision_for_near_implicit_account(
            PROTOCOL_VERSION,
            near_implicit_account_signer
        ),
```
Import PROTOCOL_VERSION From near_primitives::version::PROTOCOL_VERSION